contracts/Staking.sol

```diff
@@ -295,26 +316,36 @@ contract Staking is IStaking, Initializable, OwnableUpgradeable, Constants {
         UnbondType _type
     ) external {
         require(msg.sender == settings.getStakingManager() || msg.sender == address(this), 'G008');
-        require(unbondingLength[_source] - withdrawnLength[_source] <= 20, 'S006');
-        if(_type != UnbondType.Commission){
+        uint256 nextIndex = unbondingLength[_source];
+        if (_type == UnbondType.Undelegation) {
+            require(nextIndex - withdrawnLength[_source] < 19, 'S006');
```

Comment:
Use var such as `maxUnbondingRequest` to manage the max records. Also need to provide a set function to modify it

Reply:
yes, we can, but it need to re-depoly, not just upgrade.

Reply:
Should still be able to upgrade if we only add a new state and this state doesn't use with old block data.

Reply:
OK